pool/pool.h

```diff
@@ -98,7 +98,14 @@ struct m0_pool {
 };
 
 /**
- * Pool version is the subset of devices from the filesystem.
+ * Pool version is the subset of devices from the filesystem
+ * configuration: racks, controllers, disks, etc. If represent
+ * the whole cluster hardware configuration as a tree, the pool
+ * version would be its sub-tree.
```

Comment:
If the whole cluster hardware configuration is represented as a tree, the pool version would be its sub-tree.

Reply:
Accepted, thanks.